[Doc] Update the doc for nightly build (#2390)
skypilot-org/skypilot: docs/source/getting-started/installation.rst

the pip extras above (e.g., :code:`"[aws,gcp]"`) to reduce the
dependencies installed.

> To get the latest features/updates, You may also install SkyPilot with the nightly build:
> 
> .. code-block:: console
> 
>   $ pip install -U "skypilot-nightly[all]"
> 
> Or from source:

Reviewer: ```suggestion
To get the latest features/updates, either install the nightly build:

.. code-block:: console

  $ pip install -U "skypilot-nightly[all]"

or install from source:
```
